packaging/create_rpm.sh

```diff
@@ -165,7 +167,7 @@ print_line "Creating rpm..."
 RPMLOC=$(fpm \
   -s dir \
   -t rpm \
-  -n cloudify-manager-install \
+  -n cloudify-manager-install-${COMMUNITY_OR_PREMIUM} \
```

Comment:
This shouldn't be necessary. If you look in Jenkins, you'll see that `PRERELEASE` already has `community` in it (in community versions). You can see that final filename is:
`cloudify-manager-install-17.11.22-community.x86_64.rpm`